ebu_tt_live/ui/user_input_producer/user_input_producer.html

```diff
@@ -5,25 +5,32 @@
       <script src="https://code.jquery.com/jquery-3.1.0.min.js"></script>
       <script src="template/user_input_producer_template.js"></script>
       <script src="nunjucks.js"></script>
+      <script src="uip.js"></script>
```

Comment:
this should probably be loaded in the footer, to ensure it is only loaded after the document has fully loaded.